docs/how-to-guides/software-management/releases/isis-release-process.md

```diff
@@ -40,60 +63,77 @@ Check the [AWS CodeBuild test results](https://us-west-2.codebuild.aws.amazon.co
-    - [ ] Submit a Pull Request: Submit a pull request into the dev branch. If a release branch exists, create another PR there as well.
+    - [ ] Submit a Pull Request: Submit a pull request into the `dev` branch. If a release branch exists, create another PR there as well.
 
 === "LR"
 
     - [ ] Update the Changelog: Update the release date on the version to be released in the changelog. See comments on the  [CHANGELOG.md](https://raw.githubusercontent.com/DOI-USGS/ISIS3/dev/CHANGELOG.md) for instructions. 
     - [ ] Update `code.json` by adding a new entry with the new version number and the date last modified.   
     - [ ] **Update the Authors List**:  If there are any new contributors to the project since the last release, update the `AUTHORS.rst` file to include them.
-    - [ ] Submit a Pull Request: Submit a pull request into the dev branch. If a release branch exists, create another PR there as well. 
+    - [ ] Submit a Pull Request: Submit a pull request into the `dev` branch. If a release branch exists, create another PR there as well. 
 
 === "LTS"
 
     - [ ] Update the Changelog. Move **only the bug fixes** under this release. Follow the instructions in [CHANGELOG.md](https://raw.githubusercontent.com/DOI-USGS/ISIS3/dev/CHANGELOG.md) for how to do this.
     - [ ] Update `code.json` by adding a new entry with the LTS version. e.g. an 8.0.0 LTS would be released as 8.0 (no LTS in the version name).
     - [ ] **Update the Authors List**:  If there are any new contributors to the project since the last release, update the `AUTHORS.rst` file to include them.
-    - [ ] Submit a Pull Request: Submit a pull request into the release branch. 
+    - [ ] Submit a Pull Request: Submit a pull request into the LTS release feeder branch (i.e, `9-lts`). 
+
+=== "Prod"
+
+    - [ ] Update the Changelog. Move **only bug fixes and non-breaking features** under this release. Follow the instructions in [CHANGELOG.md](https://raw.githubusercontent.com/DOI-USGS/ISIS3/dev/CHANGELOG.md) for how to do this.
+    - [ ] Update `code.json` by adding a new entry with the Prod version.
+    - [ ] **Update the Authors List**:  If there are any new contributors to the project since the last release, update the `AUTHORS.rst` file to include them.
+    - [ ] Submit a Pull Request: Submit a pull request into the Prod release feeder branch (i.e, `9-prod`). 
 
 
 !!! Success "" 
     
-    Move on to step 4 after **merging PR(s)**. 
+    Move on to step 4 after **merging PR(s)** to the feeder branch. 
 
-### 4. **Create/setup git branch**
+### 4. **Create/Setup Release Branch**
 
 Clone the repo locally with git clone. 
 
 === "RC" 
 
-    - [ ] Create a branch from dev with `x.x.x_RCy` where `x.x.x` is the version and y (e.g. `8.1.2_RC1` or `5.4.1_RC3`). 
-        * Example: `git checkout -b 8.1.2_RC1 upstream/dev`.
+    - [ ] Create an RC branch from `dev` with `x.x.x_RCy` (i.e, `8.0.0_RC1`). 
```

Comment:
Similar to previous comments, this is true for the first RC, but sebsequent RCs should likely be based off the previous